### PR TITLE
Update Private Internet Access

### LINKED ISSUE #9894: Update Private Internet Access Icon
### Brand Name

Private Internet Access

### Official Resources for Icon and Color

They got a new logo in 2021:
https://www.privateinternetaccess.com/blog/say-hello-new-improved-private-internet-access/

SVG is available in the homepage header:
https://www.privateinternetaccess.com/

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
image

**Issue:** closes #9894

**Similarweb rank:** [43990](https://similarweb.com/website/privateinternetaccess.com)

### Checklist
- [ ] I updated the JSON data in `_data/simple-icons.json`
- [ ] I optimized the icon with SVGO or SVGOMG
- [ ] The SVG `viewbox` is `0 0 24 24`

### Description
Icon & colour from [SVG](https://assets-cms.privateinternetaccess.com/img/frontend/pia_menu_logo_light.svg) in [source](https://www.privateinternetaccess.com) header.

### CODE REVIEW

**Review by adamrusted — CHANGES_REQUESTED** [review]
I'm seeing a _slight_ mis-match on this, @PeterShaggyNoble.
Could you have another look please?
image

**Review by adamrusted — APPROVED** [review]

**Review by adamrusted — CHANGES_REQUESTED** [review]
image

Still seeing a bit of a difference in the rounded shapes at the top @PeterShaggyNoble

**Comment by PeterShaggyNoble** [comment]
Only getting back around to this one now, @adamrusted, and I'm not seeing the difference you mentioned above. Will request further reviews, though, to be sure.

**Comment by service-paradis** [comment]
I'm seeing the difference stated by @adamrusted, but it is very subtle on my side:
image


There is also a tiny difference inside the keyhole:
image

It might be small enough to let this go...

File: icons/privateinternetaccess.svg
Comment (PratyushJayachandran) [inline icons/privateinternetaccess.svg:1]: I see [some differences too!!!](https://github.com/simple-icons/simple-icons/assets/35000807/11172072-2167-4049-b541-6677e3e4d7b3)

Comment (PeterShaggyNoble) [reply icons/privateinternetaccess.svg:1]: That just looks like an anti-aliasing issue between the white & the black to me, @PratyushJayachandran.

**Pull request merged**